bagitobjecttransfer/recordtransfer/static/recordtransfer/js/base/forms.js

```diff
@@ -413,12 +459,46 @@ $(() => {
     $('.add-form-row', '#transfer-form').on('click.transfer-form', (event) => {
         setupSelectOtherToggle(['.rights-select-other'], removeOther);
     })
+
+    /***************************************************************************
+     * Group Description Display
+    **************************************************************************/
+    // Add a change event listener to the group selection field
+    const selectField = $('#' + ID_SUBMISSION_GROUP_SELECTION);
+    selectField.on('change', updateGroupDescription);
+    
+    initializeGroupTransferForm();
```

Comment:
I'm still seeing an error in the console, maybe try only running this code when that variable exists, like this:

```suggestion
    if (typeof ID_SUBMISSION_GROUP_SELECTION !== 'undefined') {
        const selectField = $('#' + ID_SUBMISSION_GROUP_SELECTION);
        selectField.on('change', updateGroupDescription);
        initializeGroupTransferForm();
    }
```

Reply:
This works! I've applied your suggestion here: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/eff87b112dc8234775c6f7ae623db8fed1808561. However, I do think that it would be a good idea to:

- split the relevant JS for each form step into separate files and load them where needed (and)
- remove `forms.js` from `static/recordtransfer/js/base`

This way, only the relevant JS is loaded for each page, and we don't have to avoid running portions of JS if constants are missing, since we expect them to be there.

Reply:
Feel free to make that change, that makes sense to me

Reply:
I think it would be better to do this in a separate issue. I'll open one up for this.